extensions/data-workspace/package.json

```diff
@@ -9,7 +9,7 @@
   "icon": "images/extension.png",
   "aiKey": "AIF-37eefaf0-8022-4671-a3fb-64752724682e",
   "engines": {
-    "vscode": "*",
+    "vscode": ">=1.48.0",
```

Comment:
FYI - this was added since VS Code requires having a specified version for this. I chose the sync point of ADS at 1.25.0.

This doesn't necessarily need to be kept up with the azdata engine version though - you only really need to update it if you're using any new functionality added in a later release. But it wouldn't hurt to bump this every time you bump the azdata engine version just to be safer.